Travis builds invalid Karaf distributions

* Get a snapshot version of the Roboconf agent.
* Start it.

The **karaf.log** file contains...

```
2015-09-28 17:37:53,579 | ERROR | pool-1-thread-1  | BootFeaturesInstaller            | 6 - org.apache.karaf.features.core - 4.0.0 | Error installing boot feature repository file:
/home/travis/build/roboconf/roboconf-platform/karaf/roboconf-karaf-dist-agent/target/assembly/system/org/apache/karaf/features/framework/4.0.0/framework-4.0.0-features.xml
java.io.IOException: /home/travis/build/roboconf/roboconf-platform/karaf/roboconf-karaf-dist-agent/target/assembly/system/org/apache/karaf/features/framework/4.0.0/framework-4.0.0
-features.xml (Aucun fichier ou dossier de ce type) : file:/home/travis/build/roboconf/roboconf-platform/karaf/roboconf-karaf-dist-agent/target/assembly/system/org/apache/karaf/fe
atures/framework/4.0.0/framework-4.0.0-features.xml
	at org.apache.karaf.features.internal.service.RepositoryImpl.load(RepositoryImpl.java:85)[6:org.apache.karaf.features.core:4.0.0]
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.loadRepository(FeaturesServiceImpl.java:391)[6:org.apache.karaf.features.core:4.0.0]
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.addRepository(FeaturesServiceImpl.java:411)[6:org.apache.karaf.features.core:4.0.0]
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.addRepository(FeaturesServiceImpl.java:402)[6:org.apache.karaf.features.core:4.0.0]
	at org.apache.karaf.features.internal.service.BootFeaturesInstaller.installBootFeatures(BootFeaturesInstaller.java:86)[6:org.apache.karaf.features.core:4.0.0]
	at org.apache.karaf.features.internal.service.BootFeaturesInstaller.start(BootFeaturesInstaller.java:76)[6:org.apache.karaf.features.core:4.0.0]
	at org.apache.karaf.features.internal.osgi.Activator.doStart(Activator.java:257)[6:org.apache.karaf.features.core:4.0.0]
	at org.apache.karaf.util.tracker.BaseActivator.run(BaseActivator.java:233)[6:org.apache.karaf.features.core:4.0.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)[:1.7.0_79]
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)[:1.7.0_79]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)[:1.7.0_79]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)[:1.7.0_79]
	at java.lang.Thread.run(Thread.java:745)[:1.7.0_79]
Caused by: java.io.FileNotFoundException: /home/travis/build/roboconf/roboconf-platform/karaf/roboconf-karaf-dist-agent/target/assembly/system/org/apache/karaf/features/framework/
4.0.0/framework-4.0.0-features.xml (Aucun fichier ou dossier de ce type)
	at java.io.FileInputStream.open(Native Method)[:1.7.0_79]
	at java.io.FileInputStream.<init>(FileInputStream.java:146)[:1.7.0_79]
	at java.io.FileInputStream.<init>(FileInputStream.java:101)[:1.7.0_79]
	at sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90)[:1.7.0_79]
	at sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188)[:1.7.0_79]
	at java.net.URL.openStream(URL.java:1037)[:1.7.0_79]
	at org.apache.karaf.features.internal.service.RepositoryImpl.load(RepositoryImpl.java:80)[6:org.apache.karaf.features.core:4.0.0]
	... 12 more
2015-09-28 17:37:53,859 | INFO  | pool-1-thread-1  | FeaturesServiceImpl              | 6 - org.apache.karaf.features.core - 4.0.0 | Adding features: aries-blueprint/[4.0.0,4.0.0]
, config/[4.0.0,4.0.0], bundle/[4.0.0,4.0.0], shell-compat/[4.0.0,4.0.0], feature/[4.0.0,4.0.0], roboconf-agent/[0.5.0.SNAPSHOT,0.5.0.SNAPSHOT], instance/[4.0.0,4.0.0], log/[4.0.0
,4.0.0], ipojo-all/[1.12.1,1.12.1], ssh/[4.0.0,4.0.0]

```

... which results in...

```
2015-09-28 17:37:56,079 | ERROR |  pool-1-thread-1 | agent                            | 8 - net.roboconf.agent - 0.5.0.SNAPSHOT | [ERROR]  : [Roboconf - Default Agent] The callbac
k method start has thrown an exception : org/osgi/framework/FrameworkUtil
java.lang.NoClassDefFoundError: org/osgi/framework/FrameworkUtil
	at net.roboconf.messaging.api.reconfigurables.ReconfigurableClient.lookupMessagingClientFactoryRegistryService(ReconfigurableClient.java:96)
	at net.roboconf.messaging.api.reconfigurables.ReconfigurableClient.<init>(ReconfigurableClient.java:61)
	at net.roboconf.messaging.api.reconfigurables.ReconfigurableClientAgent.<init>(ReconfigurableClientAgent.java:44)
	at net.roboconf.agent.internal.Agent.__M_start(Agent.java:106)
	at net.roboconf.agent.internal.Agent.start(Agent.java)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[:1.7.0_79]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)[:1.7.0_79]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[:1.7.0_79]
	at java.lang.reflect.Method.invoke(Method.java:606)[:1.7.0_79]
	at org.apache.felix.ipojo.util.Callback.call(Callback.java:233)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.util.Callback.call(Callback.java:193)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.handlers.lifecycle.callback.LifecycleCallback.call(LifecycleCallback.java:86)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.handlers.lifecycle.callback.LifecycleCallbackHandler.__M_stateChanged(LifecycleCallbackHandler.java:162)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.handlers.lifecycle.callback.LifecycleCallbackHandler.stateChanged(LifecycleCallbackHandler.java)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.InstanceManager.setState(InstanceManager.java:560)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.InstanceManager.start(InstanceManager.java:440)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.ComponentFactory.createInstance(ComponentFactory.java:179)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.IPojoFactory.createComponentInstance(IPojoFactory.java:319)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.IPojoFactory.createComponentInstance(IPojoFactory.java:240)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.extender.internal.linker.ManagedType$InstanceSupport$1.call(ManagedType.java:312)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.extender.internal.linker.ManagedType$InstanceSupport$1.call(ManagedType.java:306)[27:org.apache.felix.ipojo:1.12.1]
	at org.apache.felix.ipojo.extender.internal.queue.JobInfoCallable.call(JobInfoCallable.java:114)[27:org.apache.felix.ipojo:1.12.1]
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)[:1.7.0_79]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)[:1.7.0_79]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)[:1.7.0_79]
	at java.lang.Thread.run(Thread.java:745)[:1.7.0_79]
Caused by: java.lang.ClassNotFoundException: org.osgi.framework.FrameworkUtil not found by net.roboconf.agent-monitoring-api [11]
	at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1558)
	at org.apache.felix.framework.BundleWiringImpl.access$400(BundleWiringImpl.java:79)
	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:1998)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)[:1.7.0_79]
	... 26 more
```

When built locally, I never had this error.
Can anyone try to build Roboconf locally and see if the same error also occurs?

--- a/pom.xml
+++ b/pom.xml
@@ -92,7 +92,7 @@
 	<properties>
 		<jersey.stack.version>1.18.2</jersey.stack.version>
 		<jackson.stack.version>2.3.1</jackson.stack.version>
-		<karaf.version>4.0.0</karaf.version>
+		<karaf.version>4.0.1</karaf.version>
 		<ipojo.version>1.12.1</ipojo.version>
 		<jclouds.version>1.8.1</jclouds.version>
 		<mockito.version>1.10.19</mockito.version>

